gemrb/core/GUI/GameControl.cpp

```diff
@@ -608,7 +611,9 @@ void GameControl::DrawSelf(Region screen, const Region& /*clip*/)
 	Point gameMousePos = GameMousePos();
 	// draw reticles
 	if (isFormationRotation) {
-		double angle = AngleFromPoints(gameMousePos, gameClickPoint);
+		double angle = formationBaseAngle;
+		if (Distance (gameMousePos, gameClickPoint) > 5)
```

Comment:
This 5px dead zone appears twice, so it'd be nice if you created a named constant for it, with a comment on top.